modules/serverbidBidAdapter.js

```diff
@@ -173,6 +191,12 @@ const ServerBidAdapter = function ServerBidAdapter() {
   };
 };
 
+ServerBidAdapter.createNew = function() {
+  return new ServerBidAdapter();
+};
+
 adaptermanager.registerBidAdapter(new ServerBidAdapter(), 'serverbid');
+adaptermanager.aliasBidAdapter('serverbid', 'connectad');
+adaptermanager.aliasBidAdapter('serverbid', 'oft-media');
```

Comment:
`oftmedia` is already a bid adapter alias of AppNexus. Should that be removed? It's confusing.